[INFO   ] Logger: Record log in /storage/emulated/0/Documents/kivy_proj/simple_calc_15/.kivy/logs/kivy_23-10-29_17.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  6 2021, 01:34:26) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 V@0530.0 (GIT@cac6e6f805, I5187d04b75, 1637574833) (Date:11/22/21)'>
[INFO   ] GL: OpenGL vendor <b'Qualcomm'>
[INFO   ] GL: OpenGL renderer <b'Adreno (TM) 642L'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] Lang: The file /storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv is loaded multiples times, you might have unwanted behaviors.
[WARNING] Factory: Ignored class "Ctl" re-declaration. Current -  module: None, cls: None, baseclass: Button, filename: /storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv. Ignored -  module: None, cls: None, baseclass: Button, filename: /storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv.
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv", line 29, in <module>
[WARNING] stderr:     pos: self.pis
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 32, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: AttributeError: 'Ctl' object has no attribute 'pis'
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 923, in _build_canvas
[WARNING] stderr:     value, _ = create_handler(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 245, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv", line 29:
[WARNING] stderr: ...
[WARNING] stderr:      27:						
[WARNING] stderr:      28:					Rectangle:
[WARNING] stderr: >>   29:						pos: self.pis
[WARNING] stderr:      30:						size: self.size
[WARNING] stderr:      31:				
[WARNING] stderr: ...
[WARNING] stderr: AttributeError: 'Ctl' object has no attribute 'pis'
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv", line 29, in <module>
[WARNING] stderr:     pos: self.pis
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 32, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: 
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/main.py", line 18, in <module>
[WARNING] stderr:     CalcApp().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 919, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/main.py", line 14, in build
[WARNING] stderr:     return MyLayout()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 359, in __init__
[WARNING] stderr:     self.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 463, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 541, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 663, in _apply_rule
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 663, in _apply_rule
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 663, in _apply_rule
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 596, in _apply_rule
[WARNING] stderr:     self._build_canvas(widget.canvas.before, widget,
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 929, in _build_canvas
[WARNING] stderr:     raise BuilderException(
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv", line 29:
[WARNING] stderr: ...
[WARNING] stderr:      27:						
[WARNING] stderr:      28:					Rectangle:
[WARNING] stderr: >>   29:						pos: self.pis
[WARNING] stderr:      30:						size: self.size
[WARNING] stderr:      31:				
[WARNING] stderr: ...
[WARNING] stderr: BuilderException: Parser: File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv", line 29:
[WARNING] stderr: ...
[WARNING] stderr:      27:						
[WARNING] stderr:      28:					Rectangle:
[WARNING] stderr: >>   29:						pos: self.pis
[WARNING] stderr:      30:						size: self.size
[WARNING] stderr:      31:				
[WARNING] stderr: ...
[WARNING] stderr: AttributeError: 'Ctl' object has no attribute 'pis'
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 242, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:   File "/storage/emulated/0/Documents/kivy_proj/simple_calc_15/calc.kv", line 29, in <module>
[WARNING] stderr:     pos: self.pis
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 32, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: 
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 923, in _build_canvas
[WARNING] stderr:     value, _ = create_handler(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 245, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: 
